docs/source-pytorch/common_usecases.rst

```diff
@@ -32,13 +32,6 @@ Customize and extend Lightning for things like custom hardware or distributed st
    :button_link: model/build_model.html
    :height: 100
 
-.. displayitem::
-   :header: Configure hyperparameters from the CLI
-   :description: Enable basic CLI with Lightning.
-   :col_css: col-md-12
-   :button_link: common/hyperparameters.html
-   :height: 100
-
```

Comment:
I was thinking, maybe I should keep this and only explain the bare use of argparse with Trainer, LM, etc.

Reply:
It wouldn't hurt as an educational example, but I wouldn't go deep into it. I suggest doing that in a follow-up and also have some brief examples with popular argument parsing libraries.